DuckDuckGo/GestureToolBarButton.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  BookmarksButton.swift
+//  DuckDuckGo
+//
+//  Copyright © 2019 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+protocol GestureToolBarButtonDelegate: NSObjectProtocol {
```

Comment:
I would consider matching UIKit naming of `UIToolbar` (lowercase 'b'): `GestureToolbarButtonDelegate` here and below `GestureToolBarButton`